test/maskSpec.js

```diff
@@ -69,7 +69,7 @@ describe("uiMask", function () {
       expect(scope.x).toBe("abc123");
     });
 
-	it("should not dirty or invalidate the input", function() {
+    it("should not dirty or invalidate the input", function() {
```

Comment:
Looks like this is just a bunch of tabs => spaces. Can we keep this out of the PR and make that change in another?